function-service-impl/src/main/java/com/future/function/service/impl/feature/communication/chatroom/ChatroomServiceImpl.java

```diff
@@ -138,6 +169,47 @@ public void authorizeSubscription(String userId, String chatroomId) {
       .orElseThrow(() -> new NotFoundException("Chatroom not found"));
   }
 
+  @Override
+  public void setLimitChatrooms(String userId, long limit) {
+    UriTemplate uriTemplate = new UriTemplate(redisProperties.getKey().get("limit-chatroom"));
```

Comment:
Maybe initialize at constructor instead of initializing new object every time this method is called

Reply:
Ok, will be updated in next commit